docs/how-to-guides/SPICE/creating-spice-config-files-in-spiceql.md

```diff
@@ -0,0 +1,253 @@
+
+**Example Configuration File:**
+
+Here, we'll run through an example config file, again for Messenger, encapsulating a lot of the information put into ISIS between the .db and makedb files.  
+
+.. literalinclude:: ../../SpiceQL/db/mess.json 
+   :language: JSON
+   :linenos:
+
+
+Things to note: 
+
+* the JSON format mimics ISIS's directory structure for kernels.  With the tree: 
+
+```
+   
+    <mission>.json
+     └── "<Frame>" 
+          ├── "<Binary Kernel Type>:"
+          │    └── "<Kernel Quality>:"
+          │        └── <list of regexes>       
+          ├── "<Text Kernel Type>:"
+          │     └── <list of regexes> 
+          └── "deps:"
+               └── <list of JSON pointers> 
+```
+
+* Kernel formats now use EMCAscript (i.e. regular expressions).  
+* No times are cached, these are simply computed at runtime.
+* To resolve frame chains and dependencies on other JSON keywords, the deps object can be any arbitrary pointer to another JSON block which this frame depends on for SCLKs, CKs, etc. 
+* Instead of configs going into a black box that resolves per image kernels, running a query returns JSON matching this format. 
+
+Example output for querying all kernels for Messenger: https://gist.github.com/Kelvinrr/3e383d62eead58c016b7165acdf64d60
+
+## The Flowchart™
+
+1. Identify the mission, create a new JSON file called <mission>.json, and try to stick to NAIF abbreviations (see: https://naif.jpl.nasa.gov/pub/naif/pds/data/) 
+2. Do binary kernels first (CKs, SPKs, etc.): 
+    * Look at the makedb file, translate the wildcards to EMCAscript, and place them under their respective quality keyword 
+    * If the list of regexes (or any list) has only one element, feel free to write it as a string, not an array
+    * Check for a config file, make sure the dependencies on other time-dependent kernels are handled with the `deps` keyword.  
+3. When doing text kernels: 
+    * Simply look at the `kernel.[0-9]{4}.db` file (specifically the newest one). Mimic the regexes into EMCAscript.
+    * The key is to make sure the right text kernel is going to the right instrument. For example, IKs are sometimes instrument-specific but also can be shared by all instruments on a spacecraft.  
+4. Test by creating a new gtest running `utils::listMissionKernels` and manually confirming that all kernels are accounted for. Look at FunctionalTestsSpiceQueries.cpp for some examples. 
+5. Add new config files to CMakeLists.txt's SPICEQL_CONFIG_FILES variable to make sure they get installed. 
+ 
+!!! Warning "Software In Beta"
+    As this library is a work in progress, testing these queries isn't 100% figured out, In the long term, we will test against ISIS's kernel search and make sure we get the response. 
```

Comment:
`out, In` to `out. In`?

Reply:
I should remove that whole section as it's not really in beta anymore